### PR TITLE
[PPAKKAD-26] SBT Plugin

### PR DESCRIPTION
**Pull request by salceson**

### CODE REVIEW

File: src/main/resources/MethodBang.template [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        collector ! CollectorMessage(msgWrapper.id, None, Some(actor.toString))
+        msgWrapper.msg
+      case msg =>
+        msg
+    }
+    joinPoint.proceed(Array(actor, receive, message))
+  }
+
+  @Pointcut("call(* akka.actor.ScalaActorRef.$bang(..)) && <<<ACTORS>>>")
+  def withinUnreliable(): Unit = {}
```
Comment (mariusz89016) [inline src/main/resources/MethodBang.template:48]: After 'everything' I'll change methods name - for now it's ok.

File: src/main/scala/pl/edu/agh/iet/akka_debugging/sbt/TracedActorsFinder.scala [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+import java.io.File
+
+import pl.edu.agh.iet.akka_debugging.sbt.ScalaClassDefinitionParser.parseClassDefinitions
+
+import scala.collection.mutable
+import scala.io.Source
+
+object TracedActorsFinder {
+  val TRAIT_NAME = "TracedActor"
```
Comment (mariusz89016) [inline src/main/scala/pl/edu/agh/iet/akka_debugging/sbt/TracedActorsFinder.scala]: http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods

But for it doesn't matter right now.

Comment (salceson) [inline src/main/scala/pl/edu/agh/iet/akka_debugging/sbt/TracedActorsFinder.scala]: Well - I didn't know about this convention; it's strange that they have something very different from Java (the Java convention is used here now).

I'll correct it later.

File: src/main/resources/MethodBang.template (hunk 1 above)
Comment (salceson) [inline src/main/resources/MethodBang.template:48]: If you can propose a better name, please post it, I can change it as well.

**Pull request merged**